### PR TITLE
Add fetchers for capabilities and tickets for roadmap

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Add fetchers for capabilities and tickets for roadmap in `website-content-loader` tool.

**Related issue(s)**
Resolved https://github.com/kyma-project/website/issues/277
Epic https://github.com/kyma-project/website/issues/258

### CODE REVIEW

File: tools/content-loader/src/helpers/get-unique.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const getUnique = <T>(arr: T[], comp: string) => {
```
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply tools/content-loader/src/helpers/get-unique.ts]: But I see for key in object not index

File: tools/content-loader/src/config.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ export interface CoreConfig {
 const config: CoreConfig = {
   token: process.env.APP_TOKEN || null,
   organization: process.env.APP_ORGANIZATION || "kyma-project",
-  repository: process.env.APP_REPOSITORY || "kyma",
+  repository: "",
```
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply tools/content-loader/src/config.ts]: Yes, I override it for community and docs cases in bin/main.ts file

File: tools/content-loader/README.md [hunk 3]
```diff
@@ -26,15 +26,25 @@ npm start
...
+| **APP_DOCS_BRANCHES**                   | **YES**  | `["master"]`             | The branches with documentation                             |
+| **APP_DOCS_OUTPUT**                     | **YES**  | `docs`                   | The path for storing the documentation results              |
+| **APP_DOCS_VERSIONS_CONFIG_FILE**       | **YES**  | `versions.json`          | The path to the website documentation configuration         |
+| **APP_DOCS_TEMP_DIR**                   | **YES**  | `tempDocsDir`            | The path for storing temporary data for documentation       |
+| **APP_ROADMAP_REPOSITORY**              | **YES**  | `community`              | The repository with descriptions of capabilities            |
+| **APP_ROADMAP_OUTPUT**                  | **YES**  | `roadmap`                | The path for storing the roadmap content results            |
+| **APP_ROADMAP_TEMP_DIR**                | **YES**  | `tempRoadmapDir`         | The path for storing temporary data for roadmap content     |
+| **APP_ROADMAP_CAPABILITIES_DIR**        | **YES**  | `capabilities`           | The for with capabilities                                   |
```
Comment (mmitoraj) [inline tools/content-loader/README.md]: The description is not complete.

File: tools/content-loader/README.md [hunk 4]
```diff
@@ -26,15 +26,25 @@ npm start
...
+| **APP_ZEN_HUB_TOKEN**                   | **YES**  | `null`                   | The ZenHub API OAuth token                                  |
+| **APP_ORGANIZATION**                    | **YES**  | `kyma-project`           | The GitHub organization that owns a given repository        |
+| **APP_DOCS_REPOSITORY**                 | **YES**  | `kyma`                   | The repository with documentation                           |
+| **APP_DOCS_BRANCHES**                   | **YES**  | `["master"]`             | The branches with documentation                             |
+| **APP_DOCS_OUTPUT**                     | **YES**  | `docs`                   | The path for storing the documentation results              |
+| **APP_DOCS_VERSIONS_CONFIG_FILE**       | **YES**  | `versions.json`          | The path to the website documentation configuration         |
+| **APP_DOCS_TEMP_DIR**                   | **YES**  | `tempDocsDir`            | The path for storing temporary data for documentation       |
+| **APP_ROADMAP_REPOSITORY**              | **YES**  | `community`              | The repository with descriptions of capabilities            |
```
Comment (mmitoraj) [inline tools/content-loader/README.md]:
```suggestion
| **APP_ROADMAP_REPOSITORY**              | **YES**  | `community`              | The repository with capabilities descriptions           |
```

File: tools/content-loader/README.md [hunk 5]
```diff
@@ -26,15 +26,25 @@ npm start
...
+| **APP_ROADMAP_OUTPUT**                  | **YES**  | `roadmap`                | The path for storing the roadmap content results            |
+| **APP_ROADMAP_TEMP_DIR**                | **YES**  | `tempRoadmapDir`         | The path for storing temporary data for roadmap content     |
+| **APP_ROADMAP_CAPABILITIES_DIR**        | **YES**  | `capabilities`           | The for with capabilities                                   |
+| **APP_ROADMAP_CAPABILITIES_OUTPUT**     | **YES**  | `roadmap/capabilities`   | The path for storing the capabilities results               |
+| **APP_ROADMAP_TICKETS_OUTPUT**          | **YES**  | `roadmap/tickets.json`   | The path for storing the tickets results                    |
+| **APP_ROADMAP_LABELS**                  | **YES**  | `["Epic"]`               | The labels for tickets                                      |
+| **APP_ROADMAP_NON_CATEGORIZED_ISSUES**  | **YES**  | `Future`                 | The release name for non categorized issues                 |
+| **APP_ZENHUB_URL_PREFIX**               | **YES**  | `https://app.zenhub.com/workspaces/kyma---all-repositories-5b6d5985084045741e744dea/issues` | The prefix for ZenHub issue url |
```
Comment (mmitoraj) [inline tools/content-loader/README.md]:
```suggestion
| **APP_ZENHUB_URL_PREFIX**               | **YES**  | `https://app.zenhub.com/workspaces/kyma---all-repositories-5b6d5985084045741e744dea/issues` | The prefix for a ZenHub issue url |
```

**Pull request merged**